app/config/settings.py

```diff
@@ -685,3 +685,5 @@ def strtobool(val) -> bool:
 RETINA_ADMINS_GROUP_NAME = "retina_admins"
 RETINA_IMPORT_USER_NAME = "retina_import_user"
 RETINA_EXCEPTION_ARCHIVE = "Australia"
+# Default permission types that are set for retina users on annotation models
+PERMISSION_TYPES = ("view", "add", "change", "delete")
```

Comment:
Rather than hard coding this for all models here, wouldn't it be better to get this from the `default_permissions` of the models meta class? You only ever use this to assign all, right? See `create_permissions` in `django.contrib.auth.management.__init__.py`

Reply:
Done in a390029ac705996ea10dd8ef173b8bac8bd64e36